auditlog_tests/test_settings.py

```diff
@@ -32,7 +32,7 @@
         "NAME": os.getenv("TEST_DB_NAME", "auditlog_tests_db"),
         "USER": os.getenv("TEST_DB_USER", "postgres"),
         "PASSWORD": os.getenv("TEST_DB_PASS", ""),
-        "HOST": os.getenv("TEST_DB_HOST", "127.0.0.1"),
+        "HOST": os.getenv("TEST_DB_HOST", "localhost"),
```

Comment:
Do we need this change? If so, please add it in a separate commit.

Reply:
I think that's ok, remove it.